### PR TITLE
doc: fix formatting of spawner env-vars

### PR DESCRIPTION
**Pull request by manics**
Some env-vars on https://jupyterhub.readthedocs.io/en/stable/reference/spawners.html are incorrectly formatted.

### CODE REVIEW

File: docs/source/reference/spawners.md [hunk 1]
```diff
@@ -297,8 +297,8 @@ Additional variables can be specified via the `Spawner.environment` configuratio
 
 The process environment is returned by `Spawner.get_env`, which specifies the following environment variables:
 
-- JUPYTERHUB*SERVICE_URL - the \_bind* URL where the server should launch its HTTP server (`http://127.0.0.1:12345`).
-  This includes `Spawner.ip` and `Spawner.port`; _new in 2.0, prior to 2.0 IP, port were on the command-line and only if specified_
+- JUPYTERHUB*SERVICE_URL - the \_bind\* URL where the server should launch its HTTP server (`http://127.0.0.1:12345`).
+  This includes `Spawner.ip` and `Spawner.port`; \_new in 2.0, prior to 2.0 IP, port were on the command-line and only if specified*
```
Comment (minrk) [inline docs/source/reference/spawners.md:301]: Still [renders a bit off](https://jupyterhub--4245.org.readthedocs.build/en/4245/reference/spawners.html#environment-variables-and-command-line-arguments). Maybe autoformatter's getting something mixed up with `_/*`?

```suggestion
- JUPYTERHUB_SERVICE_URL - the *bind* URL where the server should launch its HTTP server (`http://127.0.0.1:12345`).
  This includes `Spawner.ip` and `Spawner.port`; new in 2.0, prior to 2.0 IP, port were on the command-line and only if specified.
```

**Comment by minrk** [comment]
`*` seems to have been injected into JUPYTERHUB_SSL below, too.

**Comment by manics** [comment]
It's a bug in prettier so none of the above workarounds work: https://github.com/prettier/prettier/issues/12965

How do you feel about the use of `` `backticks` `` to format the env-vars as code? That's what I normally use in my own docs.

**Comment by consideRatio** [comment]
> How do you feel about the use of `backticks` to format the env-vars as code?

:+1:!

**Comment by minrk** [comment]
I somewhat prefer not using code formatting for env variables since they are already offset plenty with all caps, but if the autoformatter is confused by the interior underscores, it seems like the right fix. Go for it!

**Review by consideRatio — APPROVED** [review]

**Pull request merged**